# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../howl/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- howl-1.0.0/src/lib/howl/Posix/posix_interface.c.vanilla	2007-05-12 11:14:07.401940485 +0200
+++ howl-1.0.0/src/lib/howl/Posix/posix_interface.c	2007-05-12 11:14:25.410966761 +0200
@@ -42,6 +42,7 @@
 typedef __uint16_t 	u16;
 typedef __uint8_t		u8;
 #	include <linux/sockios.h>
+#	include <linux/types.h>
 #	include <linux/ethtool.h> 
 #endif
 
